CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ set(WW3             OFF CACHE BOOL "Enable WW3")
 set(S2S             OFF CACHE BOOL "Enable S2S")
 set(JEDI_DRIVER     OFF CACHE BOOL "Enable JEDI as top level driver")
 set(DATM            OFF CACHE BOOL "Enable Data Atmosphere")
+set(CDEPS           OFF CACHE BOOL "Enable CDEPS")
+set(CDPES_DATM      OFF CACHE BOOL "Enable CDEPS Data Atmosphere")
```

Comment:
This is a typo. It should be CDEPS_DATM

Reply:
@DeniseWorthen It will be corrected. Thanks.